lib/Fez/Util/Config.rakumod

```diff
@@ -2,16 +2,48 @@ unit module Fez::Util::Config;
 
 use Fez::Util::Json;
 
-state $CONFIG-PATH = (%*ENV<FEZ_CONFIG>//%?RESOURCES<config.json>).IO;
-state $CONFIG      = from-j($CONFIG-PATH.slurp);
+state $FALLBACK-CONFIG-PATH = %?RESOURCES<config.json>.IO;
+state %FALLBACK-CONFIG  = from-j($FALLBACK-CONFIG-PATH.slurp);
+state $USER-CONFIG-PATH = (
+  if $*DISTRO.is-win {
+    %*ENV<FEZ_CONFIG> //
+    %*ENV<APPDATA>.IO.add('fez').add('fez-config.json')
+  }
+  elsif run('xdg-user-dir', :out, :err).exitcode == 0 {
+    my $xdg-user-dir = run('xdg-user-dir', :out, :err).out.get;
```

Comment:
This relies on stuff we can't guarantee will be there.  Maybe better, how does xdg-user-dir know where?

Reply:
This is a pretty direct transfer of the relevant bits of P5s File::HomeDir. According to that module: "The official API is the xdg-user-dir executable." So checking for existence of that executable seems sensible to differentiate XDG vs traditional Unixes.